modules/installation-rhv-downloading-ansible-playbooks.adoc

```diff
@@ -25,21 +25,18 @@ $ cd playbooks
 [source,terminal,subs=attributes+]
 ----
 $  xargs -n 1 curl -O <<< '
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/bootstrap.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/common-auth.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/create-templates-and-vms.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/inventory.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/<release-version>s.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/retire-bootstrap.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/retire-<release-version>s.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/retire-workers.yml
-        https://raw.githubusercontent.com/openshift/installer/<release-version>/upi/ovirt/workers.yml'
+        https://raw.githubusercontent.com/openshift/installer/release-{product-version}/upi/ovirt/bootstrap.yml
```

Comment:
Are these links correct? They render like this:

`https://raw.githubusercontent.com/openshift/installer/release-Branch Build/upi/ovirt/bootstrap.yml`
`https://raw.githubusercontent.com/openshift/installer/release-Branch Build/upi/ovirt/common-auth.yml`
`https://raw.githubusercontent.com/openshift/installer/release-Branch Build/upi/ovirt/create-templates-and-vms.yml`
`https://raw.githubusercontent.com/openshift/installer/release-Branch Build/upi/ovirt/inventory.yml`
`https://raw.githubusercontent.com/openshift/installer/release-Branch Build/upi/ovirt/masters.yml`

Wouldn't the `Branch Build` with the space break the URL? Shouldn't it be something like `<build-branch>`?

Reply:
That's an artifact of the GitHub preview compilation, in the final publishing, the attribute {product-version} will be replaced by the current release number - 4.10 or 4.11

Reply:
see this example for the final result of the attribute - https://docs.openshift.com/container-platform/4.10/installing/installing_openstack/installing-openstack-user.html#installation-osp-downloading-playbooks_installing-openstack-user

Reply:
or this example - https://docs.openshift.com/container-platform/4.10/installing/installing_openstack/installing-openstack-user-kuryr.html#installation-osp-downloading-playbooks_installing-openstack-user-kuryr